GitExtUtils/GitUI/TaskExtensions.cs

```diff
@@ -1,200 +1,194 @@
-﻿using System;
-using System.Runtime.CompilerServices;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft;
-
-namespace GitUI
+﻿namespace GitUI
 {
-    public static class TaskExtensions
```

Comment:
@sharwell can you please confirm this class is obsolete with Microsoft.VisualStudio.Threading v16.3.52?